lite/base_certifier_test.go

```diff
@@ -1,59 +1,58 @@
-package lite_test
+package lite
 
 import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
 
+	lerr "github.com/tendermint/tendermint/lite/errors"
 	"github.com/tendermint/tendermint/types"
-
-	"github.com/tendermint/tendermint/lite"
-	liteErr "github.com/tendermint/tendermint/lite/errors"
 )
 
-func TestStaticCert(t *testing.T) {
+func TestBaseCert(t *testing.T) {
 	// assert, require := assert.New(t), require.New(t)
 	assert := assert.New(t)
 	// require := require.New(t)
 
-	keys := lite.GenValKeys(4)
+	keys := genPrivKeys(4)
 	// 20, 30, 40, 50 - the first 3 don't have 2/3, the last 3 do!
 	vals := keys.ToValidators(20, 10)
 	// and a certifier based on our known set
 	chainID := "test-static"
-	cert := lite.NewStaticCertifier(chainID, vals)
+	cert := NewBaseCertifier(chainID, 2, vals)
 
 	cases := []struct {
-		keys        lite.ValKeys
+		keys        privKeys
 		vals        *types.ValidatorSet
 		height      int64
 		first, last int  // who actually signs
 		proper      bool // true -> expect no error
 		changed     bool // true -> expect validator change error
```

Comment:
This might be leftover form the version where validator set differences were constrained to have less than 1/3 of voting power. It does not make sense in the new version.

Reply:
Now it's just "IsErrUnexpectedValidators", which is returned when the validator set is not exact.  So I believe we do need this.